parlai/core/torch_ranker_agent.py

```diff
@@ -42,6 +42,11 @@ def add_cmdline_args(cls, argparser):
                  "<cands_name> is the name of the file (not the full path) passed by "
                  "the flag --fixed-candidates-path. By default, this file is created "
                  "once and reused. To replace it, use the 'replace' option.")
+        agent.add_argument(
+            '--train-predict', type='bool', default=True,
+            help='Get predictions and calculate mean rank during the train '
+                 'step. Turning this off will speed up training.'
+        )
```

Comment:
tbh I think defaulting to False might even be better, you do have the warning for anyone who might be surprised and then we won't have to remember to set it to False every time we are training (we expect this to be the best experience for the user)